test/integration/containerized/test_container_management.py

```diff
@@ -70,3 +73,65 @@ def test_cancel_will_remove_container(test_data_dir, container_runtime_installed
     assert not is_running(
         cli, container_runtime_installed, 'ansible_runner_foo_bar'
     ), 'Found a running container, they should have all been stopped'
+
+
+def test_invalid_registry_host(tmp_path):
```

Comment:
```suggestion
@pytest.mark.skipif(shutil.which('podman') is None, reason="podman not available")
def test_invalid_registry_host(tmp_path):
```

Reply:
But as @Shrews mentioned, these should be changed to use Docker, or tested with both `podman` and `docker` skipping as appropriate.

Reply:
As an example:
```suggestion
@pytest.mark.parametrize('program', ('docker', 'podman'))
def test_invalid_registry_host(tmp_path, program):
    if shutil.which(program) is None:
        pytest.skip(f'{program} is not available')
```

Also needs to be changed on line 89:
```python
            'process_isolation_executable': program,
```

Reply:
@samdoran's suggestion to use `parametrize`  is the proper way to handle this.

Reply:
There are actually fixtures for this already, but they are all-or-nothing currently.